BraintreeDropIn/Public/BTDropInResult.h

```diff
@@ -29,6 +29,9 @@ typedef void (^BTDropInResultFetchHandler)(BTDropInResult * _Nullable result, NS
 /// The payment method nonce
 @property (nonatomic, strong, nullable) BTPaymentMethodNonce *paymentMethod;
 
+/// True if user selected a vaulted item from the BTPaymentSelectionViewController collectionView.
+@property (nonatomic, assign, getter=isVaultedItem) BOOL vaultedItem;
```

Comment:
I wonder if developers could be confused and think this value indicates whether the payment method WAS VAULTED. Maybe we can rename this to something like `selectedFromVault` or `existingPaymentMethod`. Thoughts?